.github/workflows/daily_ci.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow runs every weekday at 15:00 UTC (8AM PDT)
+name: Daily CI
```

Comment:
Just curious, is the purpose of this to catch spontaneous build breaks even when there are no active PRs? Ideally with reproducible builds this wouldn't be necessary, but I appreciate that reality can be less than ideal. :)

Reply:
I have a long standing ambition that every Crypto Tools Product executes its CI at least once every week day.
A failure of this execution would cut a Sev3 ticket.

I want this.
One Day, I will get it. 

This is a start.
It also let's us have a clean build badge.